ray-operator/apis/ray/v1alpha1/rayservice_types.go

```diff
@@ -59,10 +61,22 @@ type RayActorOptionSpec struct {
 // RayServiceStatus defines the observed state of RayService
 type RayServiceStatus struct {
 	// Important: Run "make" to regenerate code after modifying this file
-	ServiceStatus    ServiceStatus           `json:"serviceStatus,omitempty"`
-	ServeStatuses    []ServeDeploymentStatus `json:"serveDeploymentStatuses,omitempty"`
-	RayClusterName   string                  `json:"rayClusterName,omitempty"`
-	RayClusterStatus RayClusterStatus        `json:"rayClusterStatus,omitempty"`
+	ServiceStatus        ServiceStatus           `json:"serviceStatus,omitempty"`
+	ServeStatuses        []ServeDeploymentStatus `json:"serveDeploymentStatuses,omitempty"`
+	DashboardStatus      DashboardStatus         `json:"dashboardStatus,omitempty"`
+	ActiveRayClusterName string                  `json:"activeRayClusterName,omitempty"`
+	// Pending RayCluster Name indicates a RayCluster will be created or is under creating.
+	PendingRayClusterName string           `json:"pendingRayClusterName,omitempty"`
+	RayClusterStatus      RayClusterStatus `json:"rayClusterStatus,omitempty"`
```

Comment:
Can we separate ActiveRayClusterStatus and PendingRayClusterStatus?

Reply:
Sure